### PR TITLE
If service/Alert is flagged 5 times hide the service/alert

### PR DESCRIPTION
**Pull request by asmasattar**
https://github.com/Tour-de-Force/btc-app-server/issues/45

### CODE REVIEW

File: src/flag.js [hunk 1]
```diff
@@ -36,29 +36,34 @@ export default function flag( req, res ) {
 				// Point they are trying to comment on doesn't exist
 				return res.status( 400 ).json( { error: "Specified point doesn't exist"} );
 			}
-			/*Add the user name to the array flagged_by*/
+			//ELSE: there is a point and the following can take place...
 			var cur_flagged_by = point.get('flagged_by');
 
+			//chekcing to see if user already flagged the point.
```
Comment (samfalconer) [inline src/flag.js]: "chekcing" → "checking"

File: src/flag.js [hunk 2]
```diff
@@ -36,29 +36,34 @@ export default function flag( req, res ) {
...
 				}
 			}
-
+			//Add the user email and reason to the array flagged_by.
 			cur_flagged_by.push({user: req.user.email, reason: req.body.flagReason} );
 			point.set('flagged_by', cur_flagged_by);
 			point.set('updated_by', req.user.email);
-
+			
+			if ( cur_flagged_by.length == 5 ){
```
Comment (samfalconer) [inline src/flag.js]: Can we go >= 5 just to be safe here?

File: src/flag.js (hunk 1 above)
Comment (asmasattar) [reply src/flag.js]: Fixed. thank you!

File: src/flag.js (hunk 2 above)
Comment (asmasattar) [reply src/flag.js]: Yes this sounds good. I changed it to >=5.

**Pull request merged**